content/en/_index.md

```diff
@@ -6,15 +6,15 @@ menu:
   main:
     weight: 20
 no_list: true
-description: Create secure, scalable, compliant Infrastructure stack for your startup in less than an hour.
+description: Higher level Infrastructure-As-Code
 ---
 
 ## What is Opta?
 
-Opta is a new kind of Infrastructure-As-Code framework where you work with high-level constructs
-instead of getting lost in low level cloud configuration. Imagine just being able to say that you want
-an autoscaling docker container that talks to a RDS database - instead of figuring out the details of VPC,
-IAM, Kubernetes, Elastic Load Balancing etc -- that's what Opta does!
+Opta is an Infrastructure-As-Code framework where you work with high-level constructs instead of getting lost
+in low level cloud configuration. Opta gives you a vast library of modules that you can connect together to
+build your ideal Infrastructure stack. Best of all, Opta uses Terraform under the hood - so you're never locked in.
+You can always write custom Terraform or even take the Opta generated Terraform and go your own way!
```

Comment:
There is also the idea that a user might generate Terraform with opta and repeat this process, may be with some automation. Go your own ways, makes it sound that opta is no longer needed.
May something like:
Use opta to generate your Terraform stack to provision your infrastructure yourself.

Reply:
Hmm... I think we need to expand the custom-terraform module docs (https://docs.opta.dev/reference/aws/environment_modules/custom-terraform/) into a whole "Opta + Terraform" guide. And that's where we could include this info. I'll make a note of this.